unit-tests/syncer/test-ts-same-fps.py

```diff
@@ -0,0 +1,205 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+import pyrealsense2 as rs
+from rspy import log, test
+
+
+# The timestamp jumps are closely correlated to the FPS passed to the video streams:
+# syncer expects frames to arrive every 1000/FPS milliseconds!
+fps = 60
+gap = 16  # 1000 / fps
+domain = rs.timestamp_domain.hardware_clock
+
+def generate_depth_frame( frame_number, timestamp ):
```

Comment:
Redundant with generate_depth_frame on test-ts-diff-fps.py, can add syncer_common.h or something like that?

Reply:
I refactored common code into sw.py